app/dependencies.php

```diff
@@ -183,6 +184,17 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
             return new Client\LiveStripeClient($c->get(StripeClient::class));
         },
 
+        CloudWatchClient::class => static function (): CloudWatchClient {
+            return new CloudWatchClient([
+                'version' => 'latest',
+                'region' => getenv('AWS_REGION'),
```

Comment:
this is going to cause a crash any time we run the matchbot CLI in an env that doesn't have the region set in the environment, even if we aren't using the new command, so we'd have to do the infrastructure roll out first. Can we make it default to something that will be OK as placeholder as long as we don't actually need to run the stats command? Or get the env changes rolled out before merging this PR?

Reply:
I don't see why we can't roll out the infra changes from https://github.com/thebiggive/infrastructure/pull/284 ahead of this PR going in. I'll merge ahead of review now so we can spot any apply issues in Staging early.